src/messages/src_authority.rs

```diff
@@ -36,6 +36,8 @@ pub enum SrcAuthority {
     },
     /// Authority of a single peer that uses it's BLS Keyshare to sign the message.
     BlsShare {
+        /// Section name at the time
+        src_section: XorName,
```

Comment:
Should this be prefix instead?

Reply:
Talked with Adam last week about it, really no point using Prefix. From what I understood the idea was to ensure the message goes to the section it came from. But only if there's split would it go anywhere else, and that is the same with prefix as xorname - old section is not there anymore. So can just as well go with xorname.